docs/source/data_entry.rst

```diff
@@ -68,9 +68,9 @@ and :func:`~citrine.resources.data_concepts.DataConceptsCollection.filter_by_nam
 can help refine the listing to make the target object easier to find.
 
 For material runs and specs, there exist additional types of filtering:
-:func:`~citrine.resources.material_run.MaterialRunCollection.filter_by_spec`,
-:func:`~citrine.resources.material_run.MaterialRunCollection.filter_by_template`,
-and :func:`~citrine.resources.material_spec.MaterialSpecCollection.filter_by_template`.
+:func:`~citrine.resources.material_run.MaterialRunCollection.list_by_spec`,
```

Comment:
Also worth mentioning new implementations? Or maybe not if it's alpha

Reply:
I was going to but I'm not sure if it's worth the time if we're going to push for different search syntax. @djack do you want me to add this before DE takes a look?